scratch/evaluate_scheduler_qd_dense_scenario.cc

```diff
@@ -96,53 +89,53 @@ struct CommunicationPair
   Ptr<Application> srcApp;
   Ptr<PacketSink> packetSink;
   uint64_t totalRx = 0;
-  double throughput = 0;
+  Time jitter = Seconds (0);
+  Time lastDelayValue = Seconds (0);
   uint64_t appDataRate;
   Time startTime;
 };
-typedef map<Ptr<Node>, CommunicationPair> CommunicationPairList;
-typedef CommunicationPairList::iterator CommunicationPairListI;
-typedef CommunicationPairList::const_iterator CommunicationPairListCI;
+typedef std::map<Ptr<Node>, CommunicationPair> CommunicationPairList;
 
 /** Simulation Arguments **/
-string applicationType = "onoff";             /* Type of the Tx application */
-string socketType = "ns3::UdpSocketFactory";  /* Socket Type (TCP/UDP) */
-string schedulerType = "ns3::CbapOnlyDmgWifiScheduler";   /* The type of scheduler to be used */
-string phyMode = "DMG_MCS12";                 /* The MCS to be used at the Physical Layer. */
-uint32_t packetSize = 1448;                   /* Application payload size [bytes]. */
-string tcpVariant = "NewReno";                /* TCP Variant Type. */
-uint32_t maxPackets = 0;                      /* Maximum Number of Packets */
-uint32_t msduAggregationSize = 7935;          /* The maximum aggregation size for A-MSDU [bytes]. */
-uint32_t mpduAggregationSize = 262143;        /* The maximum aggregation size for A-MPDU [bytes]. */
-double simulationTime = 10;                   /* Simulation time [s]. */
-uint8_t allocationId = 1;                     /* The allocation ID of the DMG Tspec element to create */
-
-/**  Applications **/
+std::string applicationType = "onoff";             /* Type of the Tx application */
+std::string socketType = "ns3::UdpSocketFactory";  /* Socket Type (TCP/UDP) */
+std::string schedulerType = "ns3::CbapOnlyDmgWifiScheduler";   /* The type of scheduler to be used */
+std::string phyMode = "DMG_MCS12";                 /* The MCS to be used at the Physical Layer. */
+uint32_t packetSize = 1448;                        /* Application payload size [bytes]. */
+std::string tcpVariant = "NewReno";                /* TCP Variant Type. */
+uint32_t maxPackets = 0;                           /* Maximum Number of Packets */
+uint32_t msduAggregationSize = 7935;               /* The maximum aggregation size for A-MSDU [bytes]. */
+uint32_t mpduAggregationSize = 262143;             /* The maximum aggregation size for A-MPDU [bytes]. */
+double simulationTime = 10;                        /* Simulation time [s]. */
+uint8_t allocationId = 1;                          /* The allocation ID of the DMG Tspec element to create */
+uint16_t thrLogPeriodicity = 100;                  /* The log periodicity for the throughput of each STA [ms] */
```

Comment:
would improve the readability in some parts of the code
```suggestion
Time thrLogPeriodicity = MilliSeconds (100);                  /* The log periodicity for the throughput of each STA [ms] */
```